### ISSUE TITLE
Runner: passing initial custom data

### ISSUE THREAD

**Issue by Kamil93** [moleculerjs/moleculer#913]
There is `metadata` field in `config` that we can use to store custom data in broker, also runner supports setting it by process env.
But these data are shared across nodes through transport. Is there equivalent that is local only?

**Comment by intech** [comment]
For example, store a local variable?
```js
created() {
    this.myVar = process.env.myVar;
}
```

**Comment by Kamil93** [comment]
@intech Yes, but no when we are using moleculer-runner, then we have not access to broker `created` method

**Comment by AndreMaz** [comment]
Maybe async config can help you.

```js
// moleculer.config.js
const fetch = require("node-fetch");

module.exports = async function() {
    // Fetch moleculer.config
    const res = await fetch("https://pastebin.com/raw/SLZRqfHX");

    // Modify it here

    // Return
    return await res.json();
};
```

**Comment by icebob** [comment]
`export MOL_MY_VAR=myvar`
and you can access it via `broker.options.myVar`

![image](https://user-images.githubusercontent.com/306521/115578972-066d9000-a2c6-11eb-84df-652f8d04878c.png)
